docs/FAQ/gui-faq-fr.md

```diff
@@ -1,238 +1,227 @@
+DNSSEC seront effectués automatiquement.
 
 <a name="q8"></a>
-#### 8. Qu'est ce qui fait que Zonemaster est différent des outils existants ?
+#### 8. Qu'est-ce qui distingue Zonemaster des autres outils existants ?
 
-Avant toute chose, Zonemaster conserve un historique des derniers tests 
-réalisés sur une zone. Concrètement, cela signifie que vous pouvez revenir
-analyser et comparer l'état d'une zone même après avoir corrigé les problèmes
-rencontrés sur celle-ci.
+Premièrement, Zonemaster conserve tout l'historique des tests réalisés sur un
+nom de domaine, ce qui signifie que vous pouvez revenir en arrière pour
+comparer les résultats d'un test remontant à un certain temps et ceux d'un
+test que vous avez lancé il y a un instant.
 
-Zonemaster va aussi essayer de vous présenter les résultats de ces analyses,
-d'une manière lisible et compréhensible (bien que le caractère technique
-de ceux-ci peut les rendre difficiles à comprendre pour un néophyte).
+Deuxièmement, tous les tests qu'effectue Zonemaster sont définis dans des
+spécifications de cas de test, qui se trouvent dans le document intitulé
+[Defined Test Cases].
 
-Zonemaster peut réaliser des tests sur une zone qui n'est pas encore
-déléguées (plus de détails dans la réponse à la question "C'est quoi un test
-sur un nom de domaine non délégué ?" ).
+Troisièmement, Zonemaster peut être utilisé pour tester des zones non
+déléguées. Voir [Question 12].
 
-Il existe aussi la possibilité, pour les utilisateurs expérimentés, de
-passer en mode "avancé" afin d'obtenir un niveau de détails accru.
+Quatrièmement, Zonemaster peut être utilisé pour tester des enregistrements DS
+avant leur publication dans la zone parente (un prérequis pour l'activation de
+DNSSEC dans une zone signée). Voir [Question 13].
 
-Pour finir, cette version "open source" de Zonemaster a été écrite de
-manière modulaire afin de permettre son intégration en tout ou partie dans
-des systèmes existants.
+Enfin, cette version « open source » de Zonemaster a été écrite de manière
+modulaire, ce qui signifie en résumé qu'il est possible d'intégrer des parties
+de Zonemaster dans vos propres systèmes, si vous le souhaitez. Par exemple, on
+souhaite rarement d'utiliser un logiciel complet juste pour vérifier des
+redélégations.
 
 <a name="q9"></a>
-#### 9. Zonemaster et confidentialité
+#### 9. Zonemaster et le respect de la vie privée
 
-Puisque Zonemaster est accessible à tous, n'importe qui, peut vérifier
-votre nom de domaine et consulter les historiques afférents. Toutefois, il
-n'est pas possible de savoir qui a réalisé ces tests puisque cette information
-n'est pas conservée (contrairement à l'heure et au jour du test).
+Puisque Zonemaster est accessible à tous, n'importe qui peut vérifier votre
+nom de domaine et consulter son historique de tests.
+Toutefois, il n'est pas possible de savoir qui a réalisé ces tests, car seuls
+les paramètres du test et les résultats sont conservés.
```

Comment:
I think it would be better to rephrase it as: “seuls les paramètres et les résultats du test sont conservés”. Otherwise “leurs résultats” could be slightly ambiguous.